externalPackages/FMAToolbox/IO/LoadBinary.m

```diff
@@ -42,7 +42,7 @@
 % Default values
 nChannels = 1;
 precision = 'int16';
-error('this function is now deprecated, use bz_LoadBinary.m')
+%error('this function is now deprecated, use bz_LoadBinary.m')
```

Comment:
What's the reason for wanting to use the old LoadBinary? I think bz_LoadBinary has some updates - so it would be good to pick one and not have two different versions of the file floating around